source/style-guide/style/version.txt

```diff
@@ -0,0 +1,88 @@
+.. _version-numbers:
+
+==================
+Version Numbers
+==================
+
+.. default-domain:: mongodb
+
+As you document features, you often need to reference a major or specific product release version to indicate when the feature became available, changed, or was deprecated.
+
+Version references can help explain product behavior
+related to a specific product release version.
+
+With the exception of release notes and changelogs, which serve as historical
+documents, avoid referencing product release versions that we no longer support.
+For examples, when a release of MongoDB Server reaches End of Life (EOL), remove references to that release.
+
+
+You can omit information about the following items:
```

Comment:
[suggestion]

I think this sentence could be edited so it more closely ties to the previous paragraph. Consider:

"You can omit the following release-specific references: "

Reply:
Fixed